integration/ig/k8s/top_block_io_test.go

```diff
@@ -23,47 +23,80 @@ import (
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
+func newTopBlockIOCmd(ns string, cmd string, startAndStop bool) *Command {
```

Comment:
I think this is the function that we could share between `integration/ig` and `integration/inspektor-gadget`, maybe moving it to `integration/common` or so. I think it just needs to receive the `normalize` function because the rest is identical.

Notice I added two new tests to verify that (1) timeout and (2) the case when `timeout == interval` are working correctly. They should not impact the time CI is taking as both run in parallel with the one running previously.